### PR TITLE
added two functions and tests

### PR DESCRIPTION
**Pull request by mhavrylova**
Added methods for solving tasks

watermelon
nextYearFind
tests for methods

### CODE REVIEW

**Comment by pavlom0x** [comment]
Description very general
you should describe in pull request what functionality you added
mention what it should solve
### Example
Added methods for solving tasks
- watermelon
- nextYearFind
- tests for methods

File: entry-test/nextYearFind.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+'use strict';
+
```
Comment (pavlom0x) [inline entry-test/nextYearFind.js:2]: why do wi need extra space?

File: entry-test/nextYearFind.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    let count = 0;
+      for( let i = 0; i < year2.length; i++ ){        	     
+                    for( let j = 0; j < year2.length; j++ ){
+                        if (j == i) continue;
+                        if (year2[i] === year2[j] ) {
+                            count++
+                            break
+                        } 
+                    }
+        }
```
Comment (pavlom0x) [inline entry-test/nextYearFind.js]: You have indentation issue, it's bad practice write code like that

File: entry-test/nextYearFind.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+	  
+
+function nextYearFind(year){
+	let wrongNumber = 'Please enter a Number >= 1000 and <= 9000'; 
+	if ( year > 9000 || year < 1000 || year === null || year === undefined || isNaN( year ) ) {
```
Comment (pavlom0x) [inline entry-test/nextYearFind.js]: if you check `isNaN` 
this not needed `year === null || year === undefined ||`

File: entry-test/nextYearFind.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+	  return (wrongNumber);
+	}
+  	let yearN = year.toString();
```
Comment (pavlom0x) [inline entry-test/nextYearFind.js]: where you use this variable?

File: entry-test/watermelon.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+function watermelon(weight){
+   let wrongNumber = 'Please enter a Number >= 1 and <= 100';
+   let canDivideEven = 'YES';
+   let canNotDivideEven = 'NO';
+
+if ( weight < 1 || weight > 100 || weight === null || weight === undefined || isNaN( weight ) ) {
```
Comment (pavlom0x) [inline entry-test/watermelon.js]: if you check isNaN
this not needed year === null || year === undefined ||

File: entry-test/nextYearFind.js [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+                        } 
+                    }
+        }
+        if (count == 0) { 
+           return +year2.join(''); 
+        } else {
+          return nextYearFind(+year +1);
+        }    
+}
+module.exports = nextYearFind;
```
Comment (pavlom0x) [inline entry-test/nextYearFind.js]: It's good if in end of file you write empty line
in each file

**Review by pavlom0x — CHANGES_REQUESTED** [review]
Please fix indentation

File: entry-test/nextYearFind.js (hunk 1 above)
Comment (mhavrylova) [inline entry-test/nextYearFind.js]: fixed

**Review by pavlom0x — APPROVED** [review]
well done

**Comment by pavlom0x** [comment]
merge it if it ready
but you have some trouble with test `All checks have failed`

**Comment by pavlom0x** [comment]
please merge it if you done

**Pull request merged**